ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -469,6 +469,18 @@ public class BottomSheetController: UIViewController {
     }
 
     public override func viewDidLayoutSubviews() {
```

Comment:
Seems like an existing bug, but shouldn't we call super.viewDidLayoutSubviews at some point in our implementation?